File: Program_1.java
package ScreeningTest;

import java.util.Scanner;

public class Program_1 {
	public static void main(String[] args) {

	    char operator;
	    Double number1, number2, result;
	    
	    Scanner input = new Scanner(System.in);

	    // operators
	    System.out.println("Choose an operator: +, -, *, /");
	    operator = input.next().charAt(0);

	    // enter numbers
	    System.out.println("Enter first number");
	    number1 = input.nextDouble();

	    System.out.println("Enter second number");
	    number2 = input.nextDouble();

	    switch (operator) {

	      // Addition :
	      case '+':
	        result = number1 + number2;
	        System.out.println(number1 + " + " + number2 + " = " + result);
	        break;

	      // Subtraction :
	      case '-':
	        result = number1 - number2;
	        System.out.println(number1 + " - " + number2 + " = " + result);
	        break;

	      // Multiplication :
	      case '*':
	        result = number1 * number2;
	        System.out.println(number1 + " * " + number2 + " = " + result);
	        break;

	      // Division :
	      case '/':
	        result = number1 / number2;
	        System.out.println(number1 + " / " + number2 + " = " + result);
	        break;

	      default:
	        System.out.println("Invalid operator!");
	        break;
	    }

	    input.close();
	  }
}

/* Outputs Of the Program :
 
 
 
 (1) Addition :
 
	Choose an operator: +, -, *, /
	+
	Enter first number
	3
	Enter second number
	4
	3.0 + 4.0 = 7.0
 
 
 (2) Subtraction :
 
	Choose an operator: +, -, *, /
	-
	Enter first number
	5
	Enter second number
	4
	5.0 - 4.0 = 1.0
 
 
 (3) Multiplication :

	Choose an operator: +, -, *, /
	*
	Enter first number
	4
	Enter second number
	3
	4.0 * 3.0 = 12.0
 
 
 (4) Division :
 
	Choose an operator: +, -, *, /
	/
	Enter first number
	5
	Enter second number
	2
	5.0 / 2.0 = 2.5


 // Invalid Operator :
 
	Choose an operator: +, -, *, /
	1
	Enter first number
	3
	Enter second number
	4
	Invalid operator!

 
 */

File: Program_1.py
# Function to add two numbers
def add(num1, num2):
    return num1 + num2
 
# Function to subtract two numbers
def subtract(num1, num2):
    return num1 - num2
 
# Function to multiply two numbers
def multiply(num1, num2):
    return num1 * num2
 
# Function to divide two numbers
def divide(num1, num2):
    return num1 / num2
 
print("Please select operation -\n" \
        "1. Add\n" \
        "2. Subtract\n" \
        "3. Multiply\n" \
        "4. Divide\n")
 
 
# Take input from the user
select = int(input("Select operations form 1, 2, 3, 4 :"))
 
number_1 = int(input("Enter first number: "))
number_2 = int(input("Enter second number: "))
 
if select == 1:
    print(number_1, "+", number_2, "=",
                    add(number_1, number_2))
 
elif select == 2:
    print(number_1, "-", number_2, "=",
                    subtract(number_1, number_2))
 
elif select == 3:
    print(number_1, "*", number_2, "=",
                    multiply(number_1, number_2))
 
elif select == 4:
    print(number_1, "/", number_2, "=",
                    divide(number_1, number_2))
else:
    print("Invalid input")





# Outputs Of the Program :

'''
(1) Addition :

Please select operation -
1. Add
2. Subtract
3. Multiply
4. Divide

Select operations form 1, 2, 3, 4 :1
Enter first number: 3
Enter second number: 4
3 + 4 = 7


(2) Substraction :

Please select operation -
1. Add
2. Subtract
3. Multiply
4. Divide

Select operations form 1, 2, 3, 4 :2
Enter first number: 5
Enter second number: 4
5 - 4 = 1


(3) Multiplication :

Please select operation -
1. Add
2. Subtract
3. Multiply
4. Divide

Select operations form 1, 2, 3, 4 :3
Enter first number: 3
Enter second number: 4
3 * 4 = 12


(4) Division :

Please select operation -
1. Add
2. Subtract
3. Multiply
4. Divide

Select operations form 1, 2, 3, 4 :4
Enter first number: 6
Enter second number: 3
6 / 3 = 2.0


(5) Invalid Operator :

Please select operation -
1. Add
2. Subtract
3. Multiply
4. Divide

Select operations form 1, 2, 3, 4 :5
Enter first number: 4
Enter second number: 3
Invalid input

'''

